tests/publiccloud/ssh_interactive_start.pm

```diff
@@ -16,17 +16,40 @@ use Mojo::Base 'publiccloud::ssh_interactive_init';
 use publiccloud::ssh_interactive;
 use testapi;
 use utils;
+use publiccloud::utils "select_host_console";
 
 sub run {
     my ($self, $args) = @_;
-    select_console 'tunnel-console';
+
+    # This ensure that we have used the setup console, even if no module was run before.
+    select_host_console();
+    my $setup_console = current_console();
 
     # Establish the tunnel (it will stay active in foreground and occupy this console!)
+    select_console('tunnel-console');
     ssh_interactive_tunnel($args->{my_instance});
 
-    # Switch to root-console and SSH to the instance
-    # every other loaded test must stay in root-console
```

Comment:
This line is not true.

Reply:
Don't know what to say. This is how I understood it, this line: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/susedistribution.pm#L843 does the trick on **every** console expect `tunnel`.

Reply:
Exactly. This was not a suggestion - this was just a statement that this line was not saying the truth.